js/script.js

```diff
@@ -321,6 +340,23 @@ async function loadWorkflow(workflowType = null) {
     }
 }
 
+const hotkeyMap = [
+    { key: "Mod-s", run: saveFile },
+    { key: "Mod-o", run: openFile },
+    { key: "Alt-n", run: newFile },
+    { key: "Mod-r", run: saveRunFile },
```

Comment:
Would you want to do Ctrl instead of Mod on windows? Or does CodeMirror handle it?

Reply:
Code mirror handles it. It uses Cmd on a mac and Ctrl on a PC.